src/nanosvgrast.h

```diff
@@ -331,6 +331,7 @@ static float nsvg__normalize(float *x, float* y)
 }
 
 static float nsvg__absf(float x) { return x < 0 ? -x : x; }
+static float nsvg__roundf(float x) { return (x > 0) ? floorf(x + 0.5) : ceilf(x + 0.5); }
```

Comment:
Sorry for late review, but noticed this: I think the third part should be `ceilf(x - 0.5)` and not `ceilf(x + 0.5)`

Am I wrong?

Reply:
My quick godbolt agrees with you, and also >= 0 so that positive zero stays positive.

Reply:
Fixed in https://github.com/memononen/nanosvg/commit/cb0ae54e6b147ccdf85401ef3ef20f2c761252c0

Reply:
Thanks